packages/playground/blueprints/src/lib/blueprint.ts

```diff
@@ -99,9 +96,9 @@ export interface Blueprint {
 		  };
 
 	/**
-	 * The PHP extensions to use.
+	 * @deprecated No longer used. Feel free to remove it from your Blueprint.
 	 */
-	phpExtensionBundles?: SupportedPHPExtensionBundle[];
+	phpExtensionBundles?: any;
```

Comment:
This looks fine, but it made me wonder:
Have we ever considered support for dynamically loading specific, optional PHP modules?

Reply:
Yup! https://github.com/WordPress/wordpress-playground/issues/89